src/components/ground-nav/ground-nav.scss

```diff
@@ -107,10 +108,10 @@ $_ground-nav-border-color: color.$base-gray-light;
 }
 
 .c-ground-nav__menu {
-  border-block-end: $_ground-nav-border-width $_ground-nav-border-style
-    $_ground-nav-border-color;
-  border-block-start: $_ground-nav-border-width $_ground-nav-border-style
-    $_ground-nav-border-color;
+  border-color: $_ground-nav-border-color-fallback;
+  border-color: $_ground-nav-border-color;
+  border-style: $_ground-nav-border-style;
+  border-width: $_ground-nav-border-width 0;
```

Comment:
I find it easier to just set the global border styles and then toggle width. Since I'm already using longhand for the fallback color value and `border-width`, it felt silly to cling to the shorthand at all.